internal/logger/cmdline.go

```diff
@@ -7,25 +7,24 @@ SPDX-License-Identifier: AGPL-3.0-only
 package logger
 
 import (
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
+	"log/slog"
 )
 
 // CmdLineVerbosityDescription explains numeric log levels.
-const CmdLineVerbosityDescription = "log verbosity in zap logging levels. Use -1 for debug information, 0 for info, 1 for warn, 2 for error"
+const CmdLineVerbosityDescription = "log verbosity in slog logging levels. Use -4 for debug information, 0 for info, 4 for warn, 8 for error"
 
-// VerbosityFromInt converts a verbosity level from an integer to a zapcore.Level.
-func VerbosityFromInt(verbosity int) zapcore.Level {
+// VerbosityFromInt converts a verbosity level from an integer to a slog.Level.
+func VerbosityFromInt(verbosity int) slog.Level {
 	switch {
-	case verbosity <= -1:
-		return zap.DebugLevel
+	case verbosity <= -4:
+		return slog.LevelDebug
 	case verbosity == 0:
-		return zap.InfoLevel
-	case verbosity == 1:
-		return zap.WarnLevel
-	case verbosity >= 2:
-		return zap.ErrorLevel
+		return slog.LevelInfo
+	case verbosity == 4:
+		return slog.LevelWarn
+	case verbosity >= 8:
+		return slog.LevelError
```

Comment:
@msanft @daniel-weisse What do you think about this change? Is it ok or should we stay compatible with the previous log level values?